src/App.tsx

```diff
@@ -1,7 +1,33 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+interface State {
+  pressedKey: string;
+}
+
+export class App extends React.Component<{}, State> {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
+  };
+
+  componentDidMount(): void {
+    window.addEventListener('keydown', this.handleKeyboardPress);
+  }
+
+  componentWillUnmount(): void {
+    window.removeEventListener('keydown', this.handleKeyboardPress);
+  }
+
+  handleKeyboardPress = (event: KeyboardEvent): void => {
+    this.setState({ pressedKey: `The last pressed key is [${event.key}]` });
```

Comment:
So, each time you set this string?
So, why do you give the name `pressedKey` for this string, it can be `Nothing was pressed yet`?
Just put a key into the state